website/source/docs/builders/vultr.html.md

```diff
@@ -37,31 +39,29 @@ builder.
 
 -   `api_key` (string) - The Vultr API Key to access your account.
 
--   `os_id` (int) - The id of the os to use. This will be the OS that will be used to launch a new instance and provision it. See <a href="https://www.vultr.com/api/#os_os_lists" class="uri">https://www.vultr.com/api/#os_os_list</a>.
+-   `os_id` (int) - The id of the os to use. This will be the OS that will be used to launch a new instance and provision it. See [List Operating Systems](https://www.vultr.com/api/v2/#operation/list-os)
 
--   `region_id` (int) - The id of the region to launch the instance in. See
-    <a href="https://www.vultr.com/api/#regions_region_list" class="uri">https://www.vultr.com/api/#regions_region_list</a>
+-   `region_id` (string) - The id of the region to launch the instance in. See [List Regions](https://www.vultr.com/api/v2/#operation/list-regions)
     
--   `plan_id` (string) - The id of the plan you wish to use. See
-    <a href="https://www.vultr.com/api/v2/#tag/plans" class="uri">https://www.vultr.com/api/v2/#tag/plans</a>
+-   `plan_id` (string) - The id of the plan you wish to use. See [List Plans](https://www.vultr.com/api/v2/#tag/plans)
 
 ### Optional:
 
 -   `snapshot_description` (string) - Description of the snapshot.
 
--   `snapshot_id` (string) -   If you've selected the 'snapshot' (OS 164) operating system, this should be the SNAPSHOTID.
+-   `snapshot_id` (string) -   If you've selected the 'snapshot' (OS 164) operating system, this should be the ID of the snapshot [Snapshot reference](https://www.vultr.com/api/v2/#operation/list-snapshots)
 
--   `iso_id` (int) - If you've selected the 'custom' (OS 159) operating system, this is the ID of a specific ISO to mount during the deployment.
+-   `iso_id` (string) - If you've selected the 'custom' (OS 159) operating system, this is the ID of a specific ISO to mount during the deployment. [ISO reference](https://www.vultr.com/api/v2/#operation/list-isos)
 
--   `app_id` (int) - If launching an application (OSID 186), this is the APPID to launch.
+-   `app_id` (int) - If launching an application (OSID 186), this is the ID to launch. [App reference](https://www.vultr.com/api/v2/#operation/list-applications)
 
 -   `enable_ipv6` (bool) - IPv6 subnet will be assigned to the machine.
 
 -   `enable_private_network` (bool) - Enables private networking support to the new server.
 
--   `script_id` (string) - If you've not selected a 'custom' (OS 159) operating system, this can be the SCRIPTID of a startup script to execute on boot. 
+-   `script_id` (string) - If you've not selected a 'custom' (OS 159) operating system, this can be the `id` of a startup script to execute on boot. [Startup Script reference](https://www.vultr.com/api/v2/#operation/list-startup-scripts)
```

Comment:
Missing period and inconsistent with other references. To be consistent, should be:

```
this can be the `id` of a startup script to execute on boot. See [Startup Script](https://www.vultr.com/api/v2/#operation/list-startup-scripts).
```